src/app/api/shopping-list-items/shoppingListItemId/route.js

```diff
@@ -0,0 +1,200 @@
+import { NextResponse } from "next/server";
+import { getProfileIfMember } from "../../_utils";
+
+export const dynamic = "force-dynamic";
+
+/**
+ * @swagger
+ * /api/shopping-list-items/[shoppingListItemId]:
```

Comment:
Nice, we should also make the route the same, as in we can add the [] brackets to the folder name so that it's obvious it's a dynamic endpoint :))

Also is there a way to make dynamic endpoints like this api/shopping-list-items/[slug]/something -- as in a dynamic component and then something hardcoded? I couldn't figure it out earlier. I tried to do api/rooms/[roomId]/members but it didn't work :(

Reply:
oh interesting, I haven't looked into the static + dynamics + static version 🤔 maybe then just using a queryParam is better like api/rooms/members?roomID=231231231231

and re dynamic endpoint yeah you are completely right, i forgot the brackets lol